doc/classes/PhysicsDirectSpaceState2D.xml

```diff
@@ -7,7 +7,8 @@
 		Direct access object to a space in the [PhysicsServer2D]. It's used mainly to do queries against objects and areas residing in a given space.
 	</description>
 	<tutorials>
-		<link title="Ray-Casting">https://docs.godotengine.org/en/latest/tutorials/physics/ray-casting.html</link>
```

Comment:
Fixed capitalization here to match the other `title` instances for the same doc.